### ISSUE TITLE
Conveyors & Conduits do not indicate Material Flow properly at starting bridges

### ISSUE THREAD

**Issue by famo** [Anuken/Mindustry#1317]
**Platform**: *Linux*

**Build**: *steam build 102.3*

**Issue**: *Conveyors and conduits do not indicate properly the material flow at bridges who are at the starting end.*

**Steps to reproduce**:
*An Image says more than thousands words:*
![Screenshot_20200104_171919](https://user-images.githubusercontent.com/692646/71768671-84aef300-2f18-11ea-9657-3d4b8f9fa80f.png)

**Comment by Anuken** [comment]
Not a bug; bridges *can* output from any direction, which is why the conduits/conveyors connect. Tiling is not based on the current configuration of the bridge/conduit, which can change at any time.

**Comment by famo** [comment]
I'm aware that bridges _can_ output to any direction and that this technical speaking might not classify as a bug.
However it is misleading and bridge-setups are already hard to follow as it is. So a proper indication would help here and it certainly would be nice to have.
